set.mm

```diff
@@ -632959,6 +633115,92 @@ fixed reference functional determined by this vector (corresponding to
       ( cn wcel cgcd co gcdnncl mp2an ) AEFBEFABGHEFCDABIJ $.
   $}
 
+  ${
+    dvdstrd.1 $e |- ( ph -> K e. ZZ ) $.
+    dvdstrd.2 $e |- ( ph -> M e. ZZ ) $.
+    dvdstrd.3 $e |- ( ph -> N e. ZZ ) $.
+    dvdstrd.4 $e |- ( ph -> K || M ) $.
+    dvdstrd.5 $e |- ( ph -> M || N ) $.
+    $( The divides relation is transitive, a deduction version.  (Contributed
+       by metakunt, 12-May-2024.) $)
+    dvdstrd $p |- ( ph -> K || N ) $=
+      ( cdvds wbr cz wcel w3a wa wi 3jca dvdstr syl mp2and ) ABCJKZCDJKZBDJKZHI
+      ABLMZCLMZDLMZNUAUBOUCPAUDUEUFEFGQBCDRST $.
+  $}
+
+  ${
+    muldvds1d.1 $e |- ( ph -> K e. ZZ ) $.
+    muldvds1d.2 $e |- ( ph -> M e. ZZ ) $.
+    muldvds1d.3 $e |- ( ph -> N e. ZZ ) $.
+    muldvds1d.4 $e |- ( ph -> ( K x. M ) || N ) $.
+    $( If a product divides an integer, so does one of its factors, a deduction
+       version.  (Contributed by metakunt, 12-May-2024.) $)
+    muldvds1d $p |- ( ph -> K || N ) $=
+      ( cmul co cdvds wbr cz wcel w3a wi 3jca muldvds1 syl mpd ) ABCIJDKLZBDKLZ
+      HABMNZCMNZDMNZOUAUBPAUCUDUEEFGQBCDRST $.
+  $}
+
+  ${
+    muldvds2d.1 $e |- ( ph -> K e. ZZ ) $.
+    muldvds2d.2 $e |- ( ph -> M e. ZZ ) $.
+    muldvds2d.3 $e |- ( ph -> N e. ZZ ) $.
+    muldvds2d.4 $e |- ( ph -> ( K x. M ) || N ) $.
+    $( If a product divides an integer, so does one of its factors, a deduction
+       version.  (Contributed by metakunt, 12-May-2024.) $)
+    muldvds2d $p |- ( ph -> M || N ) $=
+      ( cz wcel w3a cmul co cdvds wbr 3jca muldvds2 sylc ) ABIJZCIJZDIJZKBCLMDN
+      OCDNOASTUAEFGPHBCDQR $.
+  $}
+
+  ${
+    nndivdvdsd.1 $e |- ( ph -> M e. NN ) $.
+    nndivdvdsd.2 $e |- ( ph -> N e. NN ) $.
+    $( A natural number divides a natural number if and only if the quotient is
```

Comment:
We usually call the elements of ` NN ` "positive integers", because there was a long (and never ending) dispute about ` NN ` or ` NN0 ` to be called "the natural numbers".

See also comment for ~nndivdvds: "Strong form of ~ dvdsval2 for **positive integers**."